src/exports.jl

```diff
@@ -17,6 +17,7 @@ export
     DiscontinuousLagrange,
     Serendipity,
     getnbasefunctions,
+    getrefshape,
```

Comment:
I use this regularly, so I decided to export it here. If you think that this requires more discussion then I will revert it and adjust the test.